src/app.js

```diff
@@ -728,17 +687,17 @@ function run () {
     app: self.event,
     udapp: udapp.event
   }
-  var righthandPanel = new RighthandPanel(rhpAPI, rhpEvents, {}) // eslint-disable-line
-  self._view.rightpanel.appendChild(righthandPanel.render())
-  righthandPanel.init()
+  self._components.righthandpanel = new RighthandPanel(rhpAPI, rhpEvents, {})
```

Comment:
Oh, that's a mistake :-) `filepanel` should also be `self._components.filepanel`

I agree with renaming the righthandpanel to tabspanel or maybe even in the spirit of browsers `devtoolspanel` :-)

I just try to standardize the codebase more, so that no matter where any existing or new programmer will go, he will find familiar patterns so it's easier to expect whats going on and where things are.

Maybe the **issue** where we documented "best practices" should become a markdown file in a github repository so it's possible to make PULL REQUESTS to our best practices.